src/consts.py

```diff
@@ -4,5 +4,18 @@
 """This module defines constants used throughout the Flask application."""
 
 FLASK_CONTAINER_NAME = "flask-app"
-FLASK_APP_PORT = 8000
 FLASK_SERVICE_NAME = "flask-app"
+FLASK_ENV_CONFIG_PREFIX = "FLASK_"
```

Comment:
IHMO `FLASK_ENV_CONFIG_PREFIX`and `KNOWN_CHARM_CONFIG` would be better defined in `flask_app.py` and `charm_state.py`, respectively, for higher cohesion

Reply:
Updated, thanks!